aiida_quantumespresso/calculations/pw2gw.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""`CalcJob` implementation for the pw2gw.x code of Quantum ESPRESSO."""
+from __future__ import absolute_import
+
+import os
+
+from aiida.orm import RemoteData
+from aiida.common import datastructures
+from aiida_quantumespresso.calculations.namelists import NamelistsCalculation
+
+
+class Pw2gwCalculation(NamelistsCalculation):
+    """`CalcJob` implementation for the pw2gw.x code of Quantum ESPRESSO."""
+
+    _INPUT_PSEUDOFOLDER = './pseudo/'
+
+    _EPS_X = 'epsX.dat'
+    _EPS_Y = 'epsY.dat'
+    _EPS_Z = 'epsZ.dat'
+    _EPS_TOT = 'epsTOT.dat'
+
+    _default_namelists = ['INPUTPP']
+    _internal_retrieve_list = [_EPS_X, _EPS_Y, _EPS_Z, _EPS_TOT]
+    _blocked_keywords = [
+        ('INPUTPP', 'outdir', NamelistsCalculation._OUTPUT_SUBFOLDER),
+        ('INPUTPP', 'prefix', NamelistsCalculation._PREFIX),
+    ]
+
+    @classmethod
+    def define(cls, spec):
+        # yapf: disable
+        super(Pw2gwCalculation, cls).define(spec)
+        spec.input('parent_folder', valid_type=RemoteData,
+            help='Output folder of a completed `PwCalculation`')
+
```

Comment:
There are no output nodes whatsoever. Although this is possible is this really what we want? Is there no useful information to parse from the `stdout` that we can parse and put in a `Dict` node called `output_parameters`. For example, why not parse the `walltime` and `walltime_seconds`? This will make this queryable and easily accessible in analyses. Also see a comment related to the `eps*.dat` files

Reply:
I have added the output node `output_parameters` that for now contains only the result of the `walltime` parsing.
As far as i know there shouldn't really be any information in the stdout of pw2gw that cannot be read from the original pw.x calculation or input files